lib/ClangImporter/ImportType.cpp

```diff
@@ -2847,7 +2865,21 @@ ImportedType ClangImporter::Implementation::importMethodParamsAndReturnType(
     ImportTypeKind importKind = ImportTypeKind::Parameter;
     ImportDiagnosticAdder paramAddDiag(*this, clangDecl, param->getLocation());
     Type swiftParamTy;
-    bool paramIsIUO;
+    bool paramIsIUO = false;
+    if (auto typedefType = dyn_cast<clang::TypedefType>(paramTy.getTypePtr())) {
```

Comment:
Can this and the previous addition be merged into one function or lambda?

Reply:
Good idea. I will do it in a follow-up nfc.